src/Modules/Companies/components/card.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Button, Flex, HStack, Image, Text } from "@chakra-ui/react";
+import Logo from "../../../assets/logo.svg";
+import { RiBankFill } from "react-icons/ri";
+import { TiLocationArrow } from "react-icons/ti";
+import { MdPeopleOutline } from "react-icons/md";
+import { TbArrowNarrowRight } from "react-icons/tb";
+
+const Cards = (): JSX.Element => {
+  const CardData = [1, 2, 3, 4, 5, 6];
+  return (
+    <Box
+      pt={{ base: "1rem", md: "2.5rem", lg: "3.188rem" }}
+      display="flex"
+      flexWrap="wrap"
+      gap={{ base: "1.5rem", md: "2rem", lg: "2rem" }}
+    >
+      {CardData.map((c, index) => {
```

Comment:
this component should only hold a single card, do not loop it here. the loop should be on the Companies page